commonware/response/middleware.py

```diff
@@ -43,14 +43,14 @@ class StrictTransportMiddleware(object):
     """
     Set the Strict-Transport-Security header on responses. Use the
     STS_MAX_AGE setting to control the max-age value. (Default: 1 year.)
-    Use the STS_SUBDOMAINS boolean to add includeSubdomains.
+    Use the SECURE_HSTS_INCLUDE_SUBDOMAINS boolean to add includeSubdomains.
```

Comment:
Couldn't find any reference for `STS_SUBDOMAINS`, Django docs suggests using `SECURE_HSTS_INCLUDE_SUBDOMAINS` for `includeSubdomains`